### DISCUSSION TITLE
How do I verify alerts in components via REST API ?

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#9943] [Q&A] [answered]
I would like to use the REST API to retrieve alerts info (if any) from a given project component. For instance, if there is a "Could not merge" alert

**Comment by nijel** [comment, marked as answer]
AFAIK alerts are currently not exposed in the API. You can get information when trying to update the component.

**Reply by hicksjacobp** [reply]
@nijel is there any plan to expose alerts via the REST API? It would be useful for us so that we could make Weblate statuses more obvious to engineers managing the source code repository who don't look at Weblate much.

**Reply by nijel** [reply]
There is no plan for that, but creating [a feature request](https://github.com/WeblateOrg/weblate/issues/new?assignees=&labels=&projects=&template=feature_request.yml) for that would be a good starting point.
